phpunit.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
```

Comment:
This change would need to be reverted, unless phpunit 9.3 no longer ships the schema in the package.
We use locally installed schema to ensure it always matches the version we are using

Reply:
I don't know, without this change you get a warning. This is also changed in all other PHP 8 PR's.

Reply:
@geerteltink 
I cannot reproduce the problem. The (local) schema file is there and it works in my IDE.

Reply:
phpunit throws the warning, not your ide.

Reply:
Unfortunately I cannot reproduce this either.

Reply:
Interesting... This seems to be fixed: https://github.com/sebastianbergmann/phpunit/issues/4405

So in that case we can revert this change in all other packages to `vendor/phpunit/phpunit/phpunit.xsd`.

Reply:
It is an internal implementation detail with removal reverted to keep backward compatibility.
It seems using versioned schema is now a correct way with new `--migrate-configuration` options introduced in a 9.3 series release.

So, tldr: we should treat phpunit configuration files as versioned now, with version specified in schema and use the provided migration tool.

Reply:
Also, 9.4 series released 3 weeks ago, so should probably introduce it instead of 9.3

Reply:
@Xerkus I can add `9.4` but I'm not sure there is general consensus about it. I guess there is nothing wrong using 9.3 for now.

I just followed convention in other PR's e.g. [this](https://github.com/mezzio/mezzio-swoole/pull/33/files#diff-e35810879ec42bdd81797b3ccb72f6de28a8b4a0e3bfdba43183e133e866b892R2) that was merged `6` days ago.